modules/reactor/include/meshgenerators/MeshPropertyPassing.h

```diff
@@ -0,0 +1,31 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+#include "PolygonMeshGeneratorBase.h"
+#include "MooseEnum.h"
+
+/**
+ * This MeshPropertyPassing object
+ */
+class MeshPropertyPassing : public PolygonMeshGeneratorBase
+{
```

Comment:
what is this object for?

Reply:
Please disregard this. It is just for my own testing purpose. Will not be included in the formal PR